Project/Assets/Prairie/Framework/Script/Interaction/Prompt.cs

```diff
@@ -0,0 +1,20 @@
+using UnityEngine;
+using System.Collections;
+
+public class Prompt : MonoBehaviour 
+{
+
+	public string promptText;
+	
+	public void DrawPrompt()
+	{
+		// Draw a GUI with the interaction 
+		Rect frame = new Rect (Screen.width / 2, Screen.height / 2, Screen.width / 4, Screen.height / 4); 
+
+		GUI.BeginGroup(frame); 
+			GUILayout.Box ("Click to " + promptText); 
```

Comment:
This is good for now,  but I think we shouldn't force the 'Click to'  bit,  as the user might want to Define the prompt to be just 'Open door' or 'Read book', and tell the player elsewhere what the controls are.   Always having ' click to ' would get bulky and repetitive from a game design perspective.